pages/conduct/index.tsx

```diff
@@ -12,14 +11,16 @@ const conductPage = () => (
       <title>Code of Conduct - UNIHACK</title>
     </Head>
     <Page>
-      <Header />
       <section className={styles['conduct']}>
         <div className={styles['conduct-content']}>
           <Stack size="medium">
             <HL>Code of Conduct</HL>
             <div className={styles['introduction']}>
-              <strong> We believe that every single person has the right to hack in a
-              safe and welcoming environment. </strong>
+              <strong>
+                {' '}
+                We believe that every single person has the right to hack in a
+                safe and welcoming environment.{' '}
+              </strong>
```

Comment:
here as well

Reply:
I've removed the one above the We, but not the one after environment. This is because the space is needed (but I have moved it to outside the strong tags.

Reply:
@terencehuynh Ah, I didn't know there has to be a space. All good then.